build.sbt

```diff
@@ -40,7 +40,7 @@ ThisBuild / githubWorkflowBuild := Seq(
 )
 
 ThisBuild / githubWorkflowTargetTags ++= Seq("v*")
-ThisBuild / githubWorkflowPublishTargetBranches :=
+ThisBuild / githubWorkflowPublishTargetBranches +=
```

Comment:
😄 
```suggestion
ThisBuild / githubWorkflowPublishTargetBranches ++=
```

Reply:
Yeah, I borked that completely :roll_eyes: